packages/design-system/src/components/Pill/Pill.tsx

```diff
@@ -0,0 +1,58 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2021 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+
+import { rem } from "polished";
+import styled from "styled-components";
+import { palette, spacing } from "../../styles";
+
+export type PillProps = {
+  color: string;
+  filled?: boolean;
+  textColor?: string;
+};
+
+export const pillPropsToStyles = ({
```

Comment:
probably not necessary to export this

Reply:
Great catch, Ian! Thank you so much for all of your guidance on this one. I will make the adjustment and recommit.